federator/config/types.ts

```diff
@@ -1,24 +0,0 @@
-export class ConfigChain {
```

Comment:
Why did we remove the types?

Reply:
I pute it in another file `lib/config.ts`